app.js

```diff
@@ -1 +1,84 @@
+const usersList = 
+[
+    {
+        "id":0,
+        "userName":"Tor",
+        "userAge":6456,
+        "userPhone":"0959227735",
+    },
+    {
+        "id":1,
+        "userName":"Loki",
+        "userAge":6786,
+        "userPhone":"0959223367",
+    },
+];
 
+const UserRender = ({userData})=>
+(
+    <tr>
+        <td>{userData.userName}</td>
+        <td>{userData.userAge}</td>
+        <td>{userData.userPhone}</td>
+    </tr>
+);
+
+const TableRender = ({usersData})=>
+(
+    <div>
+        <table border = "1px black">
+                <thead>
+                    <tr>
+                        <th>UserName</th>
+                        <th>UserAge</th>
+                        <th>UserPhone</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {
+                        usersData.map(user => <UserRender key={user.id} userData = {user} />)
+                    }
+               </tbody>
+        </table>
+    </div>
+);
+
+const FormForUSer =({onSubmitFunc})=>
+{
+    const[newUser,setNewValues] = React.useState(
```

Comment:
the convention is to name this pair of variables like: `x, setX`, e.g. `newUser, setNewUser` or `newValues, setNewValues`

Reply:
Done